docs/_data/table/en/user-configuration.yml

```diff
@@ -30,6 +30,9 @@ alluxio.user.file.worker.client.threads:
   How many threads to use for file worker clients to read from workers.
 alluxio.user.file.write.location.policy.class:
   The default location policy for choosing workers for writing a file's blocks
+alluxio.user.file.write.capacity.reserved.ratio:
```

Comment:
Thanks for the link, this makes more sense now. Instead of using a ratio, do you think it makes more sense for the configuration to specify a certain number of MB to reserve instead of a ratio, and have it default to 0?